site/website/preview/index.xml

```diff
@@ -16,8 +17,8 @@
   -->
 
 <page>
-  <internal-name>Site Dashboard</internal-name>
-  <content-type>/page/browse</content-type>	
-  <display-template>/templates/web/preview.ftl</display-template>
+  <internal-name>Preview</internal-name>
+  <content-type>/page/browse</content-type>
+  <display-template>/templates/web/legacy/preview.ftl</display-template>
```

Comment:
I thought this was changed to be `next` for new and `/` for old. No?

Reply:
Yes. The web url. This is just the internal structure of where the FTL is located — not something that users would perceive.

/studio/preview => /templates/web/legacy/preview.ftl
/studio/next/preview => /templates/web/next/preview.ftl